docs/_user_guide/sas.md

```diff
@@ -0,0 +1,51 @@
+# Running SAS Interactively
+
+On the Yens, SAS is available via the `module` command. 
+
+See all currently installed versions with:
+
+```title="Terminal Input"
+module avail sas
+```
+
+You will see the current SAS versions listed:
+
+```{.yaml .no-copy title="Terminal Output"}
+
+-------------------------------------------- Global Aliases --------------------------------------------
+
+
+---------------------------------------- /software/modules/Core ----------------------------------------
+   sas/9.4
+```
+
+## Load SAS Module
+To use SAS software on the Yens, load the module:
+```title="Terminal Input"
+ml sas
+```
+This will load the default `(D)` module. You can also load a specific version of SAS with:
+
+```title="Terminal Input"
+ml sas/9.4
+```
+
+## Running SAS Code
+
+To execute a SAS script at the terminal, run:
+```bash title="Terminal Input"
+sas <your-script.sas>
+```
+where `<your-script.sas>` is your SAS script.
+
+Note that SAS output is provided in a log file called `<your-script.log>`.
+
+!!! tip "Graphical SAS"
+    SAS can be also be accessed in a graphical form using [Jupyter Notebooks](/_getting_started/jupyter/#sas) or [X-Forwarding](/_user_guide/best_practices_gui).
```

Comment:
Are jupyter SAS steps necessary? That seems outdated...

Reply:
We can always update the steps on the jupyter page, but I think we should still link to it